contracts/AKAP.sol

```diff
@@ -105,6 +117,10 @@ contract AKAP is ERC721Full {
         return nodes[nodeId].nodeBody;
     }
 
+    function expireNode(uint nodeId) external onlyApproved(nodeId) {
+        nodes[nodeId].expiry = now;
+    }
+
```

Comment:
Is this away to relinquish the ownership without doing a transfer?

Reply:
Ownership doesn't change, but it makes the node available for someone else to claim.